packages/job-worker/src/playout/selectNextPart.ts

```diff
@@ -61,13 +66,58 @@ export function selectNextPart(
 		// Filter to after and find the first playabale
 		for (let index = offset; index < (length || parts.length); index++) {
 			const part = parts[index]
-			if ((!ignoreUnplayabale || isPartPlayable(part)) && (!condition || condition(part))) {
+			if (
```

Comment:
oof, this is becoming rather long and hard to read.

It looks like it could be flipped into 3 `if (...) continue` blocks instead? 
That should help with readability.